safe/gui/tools/print_report_dialog.py

```diff
@@ -590,10 +594,11 @@ def toggle_template_selector(self):
             self.template_chooser.setEnabled(False)
 
     def toggle_no_map_radio(self, checked):
-        if checked:
-            self.button_open_composer.setEnabled(False)
-        else:
-            self.button_open_composer.setEnabled(True)
+        """Slot when no map radio is clicked.
+
+        .. versionadded: 4.3.0
+        """
+        self.button_open_composer.setEnabled(not checked)
```

Comment:
So now, you can remove this slot and connect the signal to `self.button_open_composer.setDisabled` ;-)